packages/rest/__tests__/CDN.test.ts

```diff
@@ -54,14 +54,22 @@ test('discoverySplash default', () => {
 	expect(cdn.discoverySplash(id, hash)).toEqual(`${baseCDN}/discovery-splashes/${id}/${hash}.webp`);
 });
 
-test('emoji default', () => {
+test('emoji', () => {
 	expect(cdn.emoji(id)).toEqual(`${baseCDN}/emojis/${id}.webp`);
 });
 
 test('emoji gif', () => {
 	expect(cdn.emoji(id, 'gif')).toEqual(`${baseCDN}/emojis/${id}.gif`);
 });
 
+test('emoji animated', () => {
+	expect(cdn.emoji(id, { animated: true })).toEqual(`${baseCDN}/emojis/${id}.webp?animated=true`);
+});
+
+test('emoji with GIF format', () => {
+	expect(cdn.emoji(id, { extension: 'gif' })).toEqual(`${baseCDN}/emojis/${id}.gif`);
+});
```

Comment:
What happens if you provide both?

Reply:
`"https://cdn-discord.com/emojis/123456.gif?animated=true"`.

It should be noted that the typings disallow this:

```
No overload matches this call.
  Overload 1 of 2, '(emojiId: string, options?: Readonly<EmojiURLOptions> | undefined): string', gave the following error.
    Object literal may only specify known properties, and 'animated' does not exist in type 'Readonly<EmojiURLOptionsNotWebp>'.
  Overload 2 of 2, '(emojiId: string, extension?: "gif" | "webp" | "png" | "jpg" | "jpeg" | undefined): string', gave the following error.
    Argument of type '{ extension: string; animated: boolean; }' is not assignable to parameter of type '"gif" | "webp" | "png" | "jpg" | "jpeg" | undefined'.ts(2769)
```